.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+
+node_js:
+  - "6.10.1"
+  - "8.11.3"
+  - "node"
+  
+cache: yarn
+
+install:
+  - travis_retry yarn install
+
+script:
+  - yarn run lint
```

Comment:
You don't need run here, also if you want this to use yarn, you need to ensure you have a yarn.lock file in the repo.

Reply:
Its weird, but that's how travisCI determines you are "really" using yarn.

Reply:
Yeah, I just prefer using `run` to be consistent with all of the other scripts. Thanks for the `yarn.lock` file tip.